pandas/src/datetime_helper.h

```diff
@@ -20,6 +23,7 @@ npy_int64 get_long_attr(PyObject *o, const char *attr) {
   return long_val;
 }
 
+PANDAS_INLINE
 npy_float64 total_seconds(PyObject *td) {
   // Python 2.6 compat
   npy_int64 microseconds = get_long_attr(td, "microseconds");
```

Comment:
I missed that it is still used in https://github.com/pandas-dev/pandas/blob/2e276fb2fec6dd04d6abcf5e79c03853ee86cd24/pandas/src/ujson/python/objToJSON.c#L2134